codice/ddf
# DDF-4313: Unshade solr artifacts

#### What does this PR do?
Brings the DDF distribution size down from ~1.37GB to ~977MB
 - Unshaded catalog-core-solr, solr-factory, solr-xpath
 - Converted solr-factory into a service for creating clients
 - Create a bundle to export all solr related pkgs to prevent embedding
 - Removed accidentally checked in files in catalog-core-solr

#### Who is reviewing it? 

@ryeats 
@paouelle 
@figliold
@jhunzik 

#### Select relevant component teams: 
@codice/core-apis 
@codice/solr 

#### Ask 2 committers to review/merge the PR and tag them here.
@coyotesqrl
@pklinef

#### How should this be tested?

- Ensure installation works as expected
- Ensure solr can be queried with spatial, contextual and temporal queries.
 
#### Any background context you want to provide?
This a prerequisite to upgrade to Java 11

#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
